prepare-commit-msg.py

```
@@ -0,0 +1,231 @@
+        "transforms": ["middle-out"],
+        "model": model,
+        "messages": [
+            {"role": "user", "content": user_prompt},
+            {"role": "system", "content": system_prompt},
+        ],
+    }
+
+    debug_log("Making API request to OpenRouter", json.dumps(payload, indent=2))
+
+    try:
+        response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)